README.md

```diff
@@ -46,6 +46,46 @@ cargo run --example multiwindow
 
 For more information, please read the documentation below.
 
+## Rust <-> Javascript
```

Comment:
I left a comment in #39 that wish we could add descriptions to examples. Maybe you could start a `README.md` in `examples/`